connectors/aws-s3-get-object-connector/src/main/resources/camel-connector-schema.json

```diff
@@ -0,0 +1,86 @@
+{
+  "component":{
+    "kind":"component",
+    "baseScheme":"aws-s3",
+    "scheme":"aws-s3-get-object-connector",
+    "syntax":"aws-s3-get-object-connector:bucketNameOrArn",
+    "title":"S3GetObject",
+    "description":"Get specific Object with S3",
+    "label":"AWS",
+    "deprecated":false,
+    "async":false,
+    "consumerOnly":true,
+    "lenientProperties":false,
+    "javaType":"io.syndesis.s3.get.S3GetObjectConnectorComponent",
+    "groupId":"io.syndesis",
+    "artifactId":"aws-s3-get-object-connector",
+    "version":"0.5-SNAPSHOT"
+  },
+  "componentProperties":{
+    
+  },
+  "properties":{
+    "accessKey":{
```

Comment:
I know I asked in another issue (cant find it though ;-), but is accessKey / secretKey authentication the only way to secure AWS ? Or is there also an alternative like OAuth ? (like e.g. it is for the salesforce and twitter connector)

Reply:
As far as I know it's only way at the moment. 

http://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/setup-credentials.html